source/slang/slang-emit-cuda.cpp

```diff
@@ -369,6 +513,23 @@ bool CUDASourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOu
             }
             break;
         }
+        case kIROp_makeArray:
+        {
+            IRType* dataType = inst->getDataType();
+            IRArrayType* arrayType = as<IRArrayType>(dataType);
+
+            IRType* elementType = arrayType->getElementType();
+
+            // Emit braces for the FixedArray struct. 
+            m_writer->emit("{\n");
+            m_writer->indent();
+
+            _emitInitArray(elementType, inst->getOperands(), Index(inst->getOperandCount()));
```

Comment:
I might argue for this being called `_emitInitializerList` to indicate the part of the C grammar that it is emitting.